### ISSUE TITLE
Softban of IP

### ISSUE THREAD

**Issue by markmarky** [finndev/PokeBuddy#704]
I am using Pokebuddy somehow I can't find any pokestop or catch any pokemon
I tried to use my phone to login with the same wifi network. No pokestop was found.
HOWEVER, when I use my 4G network of my phone, everything comes back!! I think they are now banning the IP

**Comment by Thorben24** [comment]
Yes, they do this. You have to restart your router in this case, so you get a new ip adress. You usually only get ip banned if you are running the bot twice at the same time with 2 different accounts. Meanwhile im using a proxy tunnel each PokeBuddy instance.

**Comment by markmarky** [comment]
Any easier method to do it? Mine is school network and I can't change my ip easily

**Comment by Rekario** [comment]
@Thorben24 Are you using Proxifier for tunnel more than one instance? I thought proxifier isn´t able to tunnel more than one instance of a program throught different proxies?

**Comment by Thorben24** [comment]
@markmarky 

Uhm not sure to be honest. Can you setup a VPN on your computer? The IP ban should be temporary. Maybe try a little bit on your phone like transfer some pokemons, evolve / power up something, activate an item and after that turn back to your computer net.

@Rekario 

Yes i use. You can add more than 1 proxy and tell a specific process / instance (In this case you will have 2 x pokebuddy.exe) via a Proxification Rule in Proxifier that this tool is only allowed to use this proxy server, for the second pokebuddy instance running you do the same, just take a different proxy. 

Its also important that you dont use some public / free proxys. If you are able to buy a linux vps server with 2 ip adresses you can setup your own proxy servers like i did. I use a small vps for 5 usd @ vultr.

**Comment by Rekario** [comment]
@Thorben24 thanks for the answer. I´m using a vps with 1 adress atm. for proxifier I use some premium proxies but if I try to add for another instance another proxie it pics the first one for all because the .exe is added, not the path. Where do I think wrong? x)

**Comment by Thorben24** [comment]
@Rekario 

So you actually have 2 diferent folders on your computer, each folder you have 1 PokeBuddy setup. I dont know where you saved your folder. I have my own on my desktop, you have it maybe in your default download folder?

Anyway you have to add the PokeBuddy.exe from the first folder and give him the first proxy, and after that you create another rule and choose the .exe from your second folder. and add the second proxy.

If everything is correct it should be like my setup:
https://i.gyazo.com/50317090fcac8035fd53e0aadcd0b21b.png

**Comment by Rekario** [comment]
@Thorben24 Exactly what I´ve done. But if I have one instance running with a proxy e.g. 0.0.0.1 and add the other one e.g. 0.0.0.2, switch the second rule up both instances get the proxie-IP at the top...

**Comment by Lesepp** [comment]
What would be the port number?

**Comment by markmarky** [comment]
I have no idea about VPN or proxy. How can I do it? I just downloaded the proxifier

**Comment by markmarky** [comment]
I mean where can I find server and port?

**Comment by vendunk** [comment]
I'm receiving an IP ban after running a single instance of the bot on a single machine for 9 hours. Perhaps a reset after some time is still recommended.

**Comment by Thorben24** [comment]
@Lesepp The port number is an usual 4 digits long number who is needed for the proxy and proxifier to tunnel the traffic through this without getting interrupted. To explain how to use and install a proxy would be too much for this forum. You should google this, its not that way difficult. 

@markmarky Same for you, there are free proxy lists in the internet you can find them in google, but i can just dissuade them expressly, cause they are not stable, traffic could be intercepted and everyone who is using this proxy got the same ip adress. You should buy an own proxy server or buy a vps server and set it up by yourself.

@vendunk Strange. Maybe there is something broken atm. Idk.

**Comment by matt2310** [comment]
@Thorben24  i can buy a solid VPN for like 5$? then what my IP wont get banned?... wouldnt the one i just set up get banned tho?
